rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -19,10 +19,16 @@ string additional_constraints
 # If 'true' then the value cannot change after it has been initialized.
 bool read_only false
 
-# docs...
+# If this bounded sequence contains a floating_point_range constraint it must
+# be enforced.
```

Comment:
I don't think saying it must be enforced is correct. The user may choose to enforce it strictly, conditionally, or not at all for various scenarios.

Reply:
I'd word it more like:

> If this sequence is not empty, the constraint inside of it applies to this parameter.

And honestly, that could be said once in this file as it applies the same to all optional constraints.